MiniSim/MiniSim.swift

```diff
@@ -14,9 +14,10 @@ import UserNotifications
 class MiniSim: NSObject {
     private var menu: Menu!
     
-    @objc let statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
+    @objc var statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
```

Comment:
Why is this change needed?

Reply:
Thought the immutability affected image changes but I was incorrect, that's on me.